sdk/servicebus/service-bus/test/receiveAndDeleteMode.spec.ts

```diff
@@ -394,7 +400,9 @@ describe("Unsupported features in ReceiveAndDelete mode", function(): void {
     await afterEachTest();
   });
   async function sendReceiveMsg(testMessages: SendableMessageInfo): Promise<ServiceBusMessage> {
+    const sender = senderClient.createSender();
     await sender.send(testMessages);
+    await sender.close();
```

Comment:
Now that we don't recreate the sender and receiver clients twice in each tests, can we revert this change of creating the sender inside `sendReceiveMsg`?